source/common/filter/config_discovery_impl.h

```diff
@@ -29,6 +29,17 @@ class FilterConfigSubscription;
 
 using FilterConfigSubscriptionSharedPtr = std::shared_ptr<FilterConfigSubscription>;
 
+extern const absl::string_view HttpStatPrefix;
+extern const absl::string_view TcpListenerStatPrefix;
+extern const absl::string_view UdpListenerStatPrefix;
```

Comment:
I don't think you need extern here. Can they be defined as `constexpr` and initialized here?

Reply:
ACK